pkg/api/types.go

```diff
@@ -283,6 +283,8 @@ type LivenessProbe struct {
 	Exec *ExecAction `json:"exec,omitempty"`
 	// Length of time before health checking is activated.  In seconds.
 	InitialDelaySeconds int64 `json:"initialDelaySeconds,omitempty"`
+	// Length of time before health checking times out.  In seconds.
+	TimeoutSecs int64 `json:"timeoutSecs,omitempty"`
```

Comment:
I'm not sure that this sort of API is is appropriate for 100ms timeouts.  Or rather we should not do a bunch of engineering to make that work, at least not right now.